README.md

```diff
@@ -48,25 +97,84 @@ To disable this, you can use the `--quiet` flag to prevent output from going to
 `--no-log` flag to prevent output from going to the log file.
 
 ```
-$ topos-playground start --quiet
+$ topos-playground clean --quiet
```

Comment:
Nitpick: I think we should leave out the `--quiet` flag. If we think the default output is too verbose for regular users, then that output should be moved under a `--verbose` flag. The default should be "good for most".

Reply:
The `--quiet` flag shuts off almost all output, and the use case that I had in mind was primarily future-leaning, and is also the use case for the `--no-log` flag. Right now, the tool has limited capabilities. I can imagine that might change. If it becomes a primary tool for configuring and controlling a local devnet, there will be people who will run it over and over and over, and who might have aliases in their shell for particular ways of using it. Those people may not want it writing a log file, or generating the extra console output.

It was originally written to report almost everything to the log file, which was very frustrating to me when I first tried to use it, because when things invariably broke, that meant that I had to go diving into a separate file just to see what happened. And then, eventually, I had dozens of files hanging around. This was my original incentive to make it so that all of the informative output went to the console, too, and to make it so that the log file could be turned off.

The --quiet was added at the same time.

I'd suggest sitting on the removal of --quiet until we get a sense for the use of the tool, and the direction in which is may want to evolve.

Reply:
Ok. I fold, but I still disagree. :)